crates/sui-graphql-rpc/src/types/epoch.rs

```diff
@@ -18,6 +23,7 @@ use sui_types::sui_system_state::sui_system_state_summary::SuiSystemStateSummary
 #[derive(Clone, Debug)]
 pub(crate) struct Epoch {
     pub stored: StoredEpochInfo,
+    pub system_state: NativeSuiSystemStateSummary,
```

Comment:
Note for your follow-up PR, that you may need to make this `Option<NativeSuiSystemStateSummary>` to account for the fact that you only have this on the latest epoch.

Reply:
And actually, for this PR you may actually want to structure this as a computed field for the system state that gets `#[graphql(flattened)]` into the `Epoch` type.  This allows us to defer loading details about the system state summary unless those fields are explicitly requested, which will massively speed up other queries (e.g. if someone is just querying the epoch ID, then they load much less data).

Reply:
> account for the fact that you only have this on the latest epoch 

This slightly confuses me. Don't we want the `system_state_summary` for each epoch (so historical data too?)

Reply:
When I wrote the first message, I imagined that for historical epochs, you would be fetching system summary-related data from the `stored` field, because it will exist as a column in `epochs`, but subsequently we chatted about it, and:

- The column that holds historical system state information in `epochs` stores it as a BCS-encoded `NativeSuiSystemStateSummary` so it's fine to always expect that field to exist.
- We want to make this computed anyway, which means this field would disappear entirely, and instead we would have a computed field through which all the system state fields are accessed.